nextflow.config

```diff
@@ -34,6 +34,10 @@ profiles {
     }
 }
 
+env {
+  NXF_OPTS="-Xms1g -Xmx4g"
```

Comment:
Does this work? I have a feeling I've tried this before but as it's only set _after_ Nextflow is launched it didn't make any difference.. I may be wrong though.

Reply:
I just saw it at the CAW pipelines configuration file and thought this is neat. Didn't test it before and had to set it manually on our cluster beforehand.

Reply:
@MaxUlysse - does it work?

Reply:
I hope so...
It seems like it's working, but maybe it's just not doing anything wrong...
I'll need to test that to be sure.